packages/rocketchat-slackbridge/settings.js

```diff
@@ -15,32 +15,21 @@ Meteor.startup(function() {
 			i18nLabel: 'API_Token'
 		});
 
-		this.add('SlackBridge_AliasFormat', '', {
```

Comment:
Re-ordered the SlackBridge settings.  Put the most probably used settings toward the top